stripe/src/main/java/com/stripe/android/paymentsheet/DefaultPrimaryButton.kt

```diff
@@ -0,0 +1,95 @@
+package com.stripe.android.paymentsheet
```

Comment:
THis contains the logic now around animating and that is shared between the buy and add button